load_average/variables.tf

```diff
@@ -0,0 +1,93 @@
+variable "namespace" {
+  description = "Namespace (e.g. cp or cloudposse)."
+}
+
+variable "stage" {
+  description = "Stage (e.g. prod, dev, staging)."
+}
+
+variable "name" {
+  description = "Name (e.g. bastion or db)."
+}
+
+variable "delimiter" {
+  default = "-"
+}
+
+variable "attributes" {
+  description = "Additional attributes (e.g. policy or role)."
+  type        = "list"
+  default     = []
+}
+
+variable "tags" {
+  description = "Additional tags (e.g. map('BusinessUnit','XYZ')."
+  type        = "map"
+  default     = {}
+}
+
+variable "datadog_api_key" {
+  description = "Datadog API key. This can also be set via the DATADOG_API_KEY environment variable."
```

Comment:
If this can also be the `DATADOG_API_KEY`, then this should provide a default value of `""`